frontend/src/advising/ManageStudents.tsx

```diff
@@ -282,62 +337,139 @@ const StudentPreview = ({ id }: StudentPreviewProps) => {
+                  <IconButton onClick={() => setViewMode("editSchedule")}>
+                    <Edit />
+                  </IconButton>
+                </Tooltip>
+                <IconButton onClick={() => setViewMode("viewSchedule")}>
+                  <Fullscreen />
+                </IconButton>
+              </ButtonHeader>
+              <ScheduleWrapper>
+                <NonEditableSchedule />
+              </ScheduleWrapper>
+            </>
+          )}
+        </SchedulePreviewContainer>
+      </StudentViewContainer>
+    );
+  } else {
+    // full schedule view
+    return (
+      <ExpandedStudentPlan
+        editMode={viewMode === "editSchedule"}
+        onEditPress={() => setViewMode("editSchedule")}
+        onStopEditPress={() => setViewMode("viewSchedule")}
+        onFullScreenPress={() => setViewMode("overview")}
+      />
+    );
+  }
+};
+
+const ExpandedStudentPlan: React.FC<ExpandedStudentPlanProps> = props => {
+  const { planName, planMajor, planCoopCycle, fullName } = useSelector(
+    (state: AppState) => ({
+      planName: getActivePlanNameFromState(state),
+      planMajor: getActivePlanMajorFromState(state),
+      planCoopCycle: getActivePlanCoopCycleFromState(state),
+      fullName: getUserFullNameFromState(state),
+    })
+  );
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(expandAllYearsForActivePlanAction());
+  }, []);
+
   return (
-    <StudentPreviewContainer>
-      <StudentInfoContainer>
-        {!student && !fetchingStudent ? (
-          <NoPlanContainer>
-            <Text>User Has No Plans</Text>
-          </NoPlanContainer>
-        ) : fetchingStudent ? (
-          <LoadingSpinner />
-        ) : (
-          <StudentInfoDisplay>
-            <AvatarWrapper>
-              <Avatar>{student!.fullName[0]}</Avatar>
-            </AvatarWrapper>
-            <StudentInfoTextWrapper>
-              <NameText>{student!.fullName}</NameText>
-              <Text>{student!.email}</Text>
-              <Text>{student!.major}</Text>
-              <Text>{student!.coopCycle}</Text>
-            </StudentInfoTextWrapper>
-
-            <PlanText>Plans:</PlanText>
-            <PlanListContainer>
-              <SwitchPlanList />
-            </PlanListContainer>
-            <ButtonContainer>
-              <ColoredButton onClick={() => {}}>Assign Template</ColoredButton>
-            </ButtonContainer>
-          </StudentInfoDisplay>
-        )}
-      </StudentInfoContainer>
-      <SchedulePreviewContainer>
-        {noPlans ? (
-          <NoPlanContainer>
-            <Text>User Has No Plans</Text>
-          </NoPlanContainer>
-        ) : (
-          <>
-            <PlanTitle>
-              <TitleText>{planName}</TitleText>
-            </PlanTitle>
-            <ButtonHeader>
-              <IconButton>
-                <Edit />
+    <FullScheduleViewContainer>
+      <BackToStudentLink onClick={props.onFullScreenPress}>
+        Back to {fullName}
```

Comment:
I think this would look cleaner if this was more like a back arrow icon and in line with the student information currently being displayed below it. It feels a little awkward right now

![image](https://user-images.githubusercontent.com/49100359/103141071-32348c80-46bd-11eb-80b1-6160c6450104.png)